addons/project/models/project_project.py

```diff
@@ -208,11 +208,26 @@ def _read_group_stage_ids(self, stages, domain, order):
     milestone_count = fields.Integer(compute='_compute_milestone_count', groups='project.group_project_milestone')
     milestone_count_reached = fields.Integer(compute='_compute_milestone_reached_count', groups='project.group_project_milestone')
     is_milestone_exceeded = fields.Boolean(compute="_compute_is_milestone_exceeded", search='_search_is_milestone_exceeded')
+    milestone_progress = fields.Integer("Milestones Reached", compute='_compute_milestone_reached_count')
+    next_milestone_id = fields.Many2one('project.milestone', compute='_compute_next_milestone_id', store=True)
```

Comment:
Why store it?

Reply:
Opps! we need to make it store because we are accessing `Field.AnotherField` in depend and we can not access field via `dot` on non store field.
(P.S. Rationale from red runbot :see_no_evil: )

Reply:
Mh ok, I didn't know that!